api/src/main/resources/workbench-api.yaml

```diff
@@ -6480,6 +6480,7 @@ definitions:
     enum:
       - CROMWELL
       - RSTUDIO
+      - ALLOWED
```

Comment:
this is a pretty big PR so I'm not sure that we should also be making functional changes here.

Reply:
This should be SAS, ALLOWED is Leonardo term

Reply:
I think this diff is from outdated conflict before @yonghaoy merged his PR for adding SAS